src/test/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/AttendanceRepositoryTest.java

```diff
@@ -39,5 +39,10 @@ void getActivitiesByBookingIdAndProgramStatus() {
             Tuple.tuple(-12L, LocalDate.now(), null),
             Tuple.tuple(-11L, LocalDate.now(), null)
         );
+        assertThat(activities.getContent()).asList().extracting("eventId", "eventDate", "eventOutcome", "courseActivity.description").contains(
+            Tuple.tuple(-13L, LocalDate.of(2017, 9, 13), "UNACAB", "Woodwork"));
+        assertThat(activities.getContent()).asList().extracting("eventId", "eventDate", "eventOutcome", "courseActivity.description", "programService").contains(
+            Tuple.tuple(-12L, LocalDate.now(), null, "Woodwork", null)
```

Comment:
You will need to edit the SQL seed file to add date to avoid null values. Also Id use a single 'assertThat' which checks all fields of all 3 rows. The sql files are called something like R_?*<table name>.sql